keyboards/capsunlocked/cu80/readme.md

```diff
@@ -0,0 +1,19 @@
+# CU80
+
+The CU80 is a TKL with two hotswap PCB variants. One fixed layout ISO and one fixed layout ANSI. This single firmware tree supports both, with the `default` being ISO.
+
+* Keyboard maintainer: [rys](https://github.com/rys)
+* Hardware supported: CU80 v2
+* Hardware availability: [CapsUnlocked](https://caps-unlocked.com/cu80-round-3/)
+
+Make example for this keyboard (after setting up your build environment):
+
+ISO layout:
+
+    make capsunlocked/cu80:default
+    
+ANSI layout:
+
+    make capsunlocked/cu80:ansi
+
```

Comment:
The template has a “Flashing example for this keyboard” section here; according to the PR checklist, it should be kept and populated properly.

The PR checklist also requires “clear instructions on how to reset the board into bootloader mode” (the template does not contain that part for some reason). In this section you can mention the fact that you can hold Esc while plugging in the cable (and whether it works with the initial out-of-the-box firmware), and the location of the hardware reset button on the PCB.